pkg/graph/schema.resolvers.go

```diff
@@ -2021,11 +2024,33 @@ func (r *mutationResolver) DeleteTRBRequestFundingSources(ctx context.Context, i
 
 // SetRolesForUserOnSystem is the resolver for the setRolesForUserOnSystem field.
 func (r *mutationResolver) SetRolesForUserOnSystem(ctx context.Context, input model.SetRolesForUserOnSystemInput) (*string, error) {
-	err := r.cedarCoreClient.SetRolesForUser(ctx, input.CedarSystemID, input.EuaUserID, input.DesiredRoleTypeIDs)
+	rs, err := r.cedarCoreClient.SetRolesForUser(ctx, input.CedarSystemID, input.EuaUserID, input.DesiredRoleTypeIDs)
 	if err != nil {
 		return nil, err
 	}
 
+	// Asyncronously send an email to the CEDAR team notifying them of the change
+	go func() {
+		// make a new context, or else the request will cancel when the parent context cancels
+		emailCtx := context.Background()
+		appcontext.WithLogger(emailCtx, appcontext.ZLogger(ctx)) // copy the logger over
+
+		userInfo, err := r.service.FetchUserInfo(emailCtx, input.EuaUserID)
+		if err != nil {
+			// don't fail the request if the lookup fails, just log and return from the go func
+			appcontext.ZLogger(emailCtx).Error("failed to lookup user info for CEDAR notification email", zap.Error(err))
+			return
+		}
+
+		cedarEmailAddress := os.Getenv(appconfig.CEDAREmailAddress)
```

Comment:
I'll make the change. I also wasn't a huge fan of my `checkRequiredConfig` call for that env var being in the CEDAR Client config, rather than the email config - this is a good spot to change this as well IMO!